### ISSUE TITLE
Unexpected behaviour when ttline>0

### ISSUE THREAD

**Issue by robertbachmann** [olsak/OpTeX#121]
Given the following input:

```tex
\ttline=0

Listing 1: 
\begtt\hisyntax{C}
int main() {
  return 0;
}
\endtt

% \ttline=0

Listing 2: 
\begtt\hisyntax{C}
int foo() {
  return 0;
}
\endtt

\bye
```

I get the following result: 
![image](https://user-images.githubusercontent.com/599839/223844126-550e4cae-61ca-4a81-b0b3-fb3e17defcd2.png)

Is this intended?
Do I need to (re)set `\ttline=0` for each listing?

**Comment by olsak** [comment]
Yes, it is documented behavior. Unfortunately, numbering each listing form one is somewhat tricky:
```
\everytt{\ttline=0} \ttline=0
```
I'll think about a better solution...
